webrtc.html

```diff
@@ -1327,6 +1327,15 @@ <h4>Set the RTCSessionDescription</h4>
                     attribute set to the line number in the SDP where
                     the syntax error was detected) and abort these steps.</p>
                   </li>
+                  <li>
+                    If <var>description</var> is set as a remote description,
+                    the <var>connection</var>'s <code><a>RTCRtcpMuxPolicy</a></code>
+                    is <code><a data-link-for="RTCRtcpMuxPolicy">require</a></code>
+                    and the remote description does not use RTCP mux,
+                    then <a>reject</a> <var>p</var> with a newly <a data-link-for="exception"
+                    data-lt="create">created</a>
+                    <code>NotSupportedError</code> and abort these steps.
```

Comment:
What is the argument for using NotSupportedError and not either InvalidAccessError (thrown for all other invalid SDP) or OperationError (thrown for all other errors)?

Reply:
this is the same error as in "4.4.1.7 Set the configuration" when trying to set 'negotiate' in the constructor and this is unsupported.
I do not think this is invalidaccesserror as the SDP is valid. OperationError would wfm too

Reply:
Or InvalidStateError, which we throw when trying to perform an operation that is otherwise supported but while the PC is in an invalid state (such as being closed).

But InvalidAccessError is probably the error most consistent with other usages in the spec (rejected SDP, invalid parameters, invalid initEncodings, invalid use of addTrack/removeTrack).

NotSupportedError on the other hand is typically used when what you're doing is _not implemented by the browser_, e.g. non-muxed RTCP, url scheme name, keygenAlgorithm. This usage makes it sound like non-muxed RTCP is not implemented (like the other usage) as opposed to PC not being configured that way.

Reply:
you can't configure the PC to actually do non-muxed -- mux is either negotiate or require. And I think the usage of NotSupportedError is exactly what you say hbos :-)

Reply:
Given the comments above it feels like we should go with InvalidAccessError. Would you agree @fippo ?

Reply:
```
var pc = new RTCPeerConnection({rtcpMuxPolicy: 'require'}); pc.setRemoteDescription({type: 'offer', sdp: "v=0\r\no=- 166855176514521964 2 IN IP4 127.0.0.1\r\ns=-\r\nt=0 0\r\nm=audio 9 UDP/TLS/RTP/SAVPF 111\r\nc=IN IP4 0.0.0.0\r\na=rtcp:9 IN IP4 0.0.0.0\r\na=ice-ufrag:someufrag\r\na=ice-pwd:somelongpwdwithenoughrandomness\r\na=fingerprint:sha-256 00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00\r\na=setup:actpass\r\na=mid:audio1\r\na=sendonly\r\na=rtcp-rsize\r\na=rtpmap:111 opus/48000/2\r\n"}).catch(e => console.error(e.name))
```
throws InvalidAccessError. Not changing implementations is a good argument :-)